File: challenges/1-cubo-simples/typescript/clintonrocha/cubo-simples.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var readline = require("readline");
var rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});
rl.question("number: ", function (input) {
    var number = parseFloat(input);
    if (!isNaN(number)) {
        var cube = Math.pow(number, 3);
        console.log("".concat(number, " **3: ").concat(cube));
    }
    else {
        console.log("Invalid input.");
    }
    rl.close();
});

File: challenges/1-cubo-simples/go/ericoliveiras/cubo_simples.go
package ericoliveiras

import (
	"fmt"
	"math"
	"os"
	"strconv"
)

func main() {
	arg := os.Args[1]

	number, err := strconv.ParseFloat(arg, 64)
	if err != nil {
		fmt.Println(err)
	}

	result := math.Pow(number, 3)

	fmt.Printf("%.f³ = %.f\n", number, result)
}
